baldrick/github/github_api.py

```diff
@@ -572,6 +572,12 @@ def milestone(self):
         else:
             return milestone['title']
 
+    def get_modified_filenames(self):
+        """Get all the filenames modified by this PR."""
+        files = paged_github_json_request(self._url_files,
+                                          headers=self._headers)
+        return [f['filename'] for f in files]
```

Comment:
Can you call this ``get_modified_files``? Otherwise it sounds like just the files for which the filename has changed.

Reply:
I fixed this